src/index.html

```diff
@@ -4,24 +4,29 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Bakbak+One&display=swap" rel="stylesheet">
   <title>Leaderboard</title>
 </head>
 <body>
```

Comment:
- [ ] Good job so far 👏, but there is something that needs to be done so please kindly fix these issues. In your **README.md** file I recommend changing the usage part where you just said open your index.html file or live version. You can say run `npm install` to install packages, run `npm run build` to create dist folder, and then run `npm start` to start the app 🍀

Reply:
Thanks for the kindly suggestions @Rustamxon7